index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link rel="stylesheet" href="style.css">
+  <title>Portfolio</title>
+</head>
+<body>
+	
+	<div class="nav-container">
```

Comment:
- [ ] Kindly use the `header` tag for navbar. 😄 
_You have used the `header` tag in the wrong place._